tests/unit/test_private_test_logger.py

```diff
@@ -0,0 +1,33 @@
+import copy
+from unittest import mock
+
+from sync.private_test_logger import (PrivateTestLogger,
+                                      PrivateTestLoggerException)
+
+
+class TestPrivateTestLogger:
+    @mock.patch.object(PrivateTestLogger, '__init__', lambda *args, **kwargs: None)
+    def test_log_error(self):
+        mock_table = mock.MagicMock()
+        mock_table.put_item.return_value = {'ResponseMetadata': {'HTTPStatusCode': 404}}
+        try:
```

Comment:
If we are trying to check if the code raises an error, we should use `with self.assertRaises(Error)`